explainaboard/processors/conditional_generation.py

```diff
@@ -106,12 +106,14 @@ def _get_fre_rank(self, existing_features: dict, statistics: Any):
             existing_features, statistics, lambda x: x['source']
         )
 
-    def _complete_features(
-        self, sys_info: SysOutputInfo, sys_output: List[dict], statistics=None
-    ) -> List[str]:
-        """
-        This function is used to calculate features used for bucketing, such as sentence_length
-        :return:
+    def _gen_scoring_stats(
+        self, sys_info: SysOutputInfo, sys_output: List[dict]
+    ) -> Any:
```

Comment:
It seems the function is returning a dictionary. We could be more explicit by using dict Instead of Any.

Reply:
I debated about this, but I think theoretically the scoring stats could be in a different format. For example, a numpy array would be a good option. Maybe we can leave it as "Any" for now.

Reply:
Ah I see, then Any is fine!